sdcm/reporting/tooling_reporter.py

```diff
@@ -98,6 +102,31 @@ def _collect_version_info(self) -> None:
                 driver_version=driver_version, command_prefix=None, runner=None, argus_client=self.argus_client).report()
 
 
+class CqlStressCassandraStressVersionReporter(ToolReporterBase):
+    # pylint: disable=too-few-public-methods
+    TOOL_NAME = "cql-stress-cassandra-stress"
+
+    def _collect_version_info(self) -> None:
+        output = self.runner.run(f"{self.command_prefix} {self.TOOL_NAME} version_json")
+        LOGGER.info("%s: Collected cql-stress-cassandra-stress output:\n%s", self, output.stdout)
+        result = json.loads(output.stdout)
+
+        LOGGER.info("Result:\n%s", result)
+        cql_stress_details = result.get('cql-stress', {})
+        self.version = f"{cql_stress_details.get('version', '#FAILED_CHECK_LOGS')}"
+        self.date = cql_stress_details.get('commit_date')
+        self.revision_id = cql_stress_details.get('commit_sha')
+
+        if driver_details := result.get("scylla-driver", {}):
+            self.additional_data = f"rust-driver: {driver_details['version']}"
```

Comment:
Need to make the record be specific.
We may have more than 1 record with rust driver - one from `cql-stress` and another one from `latte`.
And those may have different versions at some point in time.

Reply:
Let's refactor this when Latte is implemented

Reply:
`additional_data` in this case is only for cql-stress, @vponomaryov  not sure what's the concern here...